tools/testnet/roles/eth_node/tasks/main.yml

```diff
@@ -0,0 +1,62 @@
+- name: Create storage fs
+  become: yes
+  filesystem:
+    dev: /dev/xvdb
+    fstype: ext4
+
+- name: Mount storage fs
+  become: yes
+  mount:
+    fstype: ext4
+    path: "{{ eth_data_dir_base }}"
+    src: /dev/xvdb
+    state: mounted
+
+- name: Create eth node data dir
+  become: true
+  file:
+    name: "{{ eth_data_dir_base }}/{{ item }}"
+    state: directory
+    owner: "{{ ansible_user }}"
+    mode: 0770
+  with_items:
+    - geth
+    - parity
+
+- name: Eth node (parity) docker container
+  docker_container:
+    name: testnet_parity
+    state: "{{ 'started' if eth_node_type == 'parity' else 'absent' }}"
+    restart_policy: always
+    pull: yes
+    image: "{{ docker_repo_eth_parity }}"
+    ports:
+      - "8545:8545"
```

Comment:
ah...regarding https://github.com/raiden-network/raiden/pull/712/files#r126714909 
wouldn't it be better to make the rpc port forwarding here to listen on a specific interface only? i.e. `- "127.0.0.1:8545:8545"`